help/en/package/jmri/jmrit/webThrottle/CamerawebThrottle.shtml

```diff
@@ -0,0 +1,102 @@
+      - it should run on Internet Explorer 10.<br>
+      <strong>There is nothing to install on your device !</strong></p>
+
+      <p>To start using JMRI through a web browser, you just need to activate the included
+      <strong>Web Server</strong>. You may found additional information <a href=
+      "/help/en/html/web/index.shtml">here</a>.</p>
+
+      <p>To get help, please click '?' at each page top right.<br>
+      You will find important and useful information there.<br>
+      There is specific Help for each page type (Roster, Loco, Panel, ...).</p>
+
+      <h3>Screen samples...</h3>
+
+      <p><img src="images/JMRI_webThrottle.png" height="300" alt="The default screen">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(loco).png" height="300" alt="A loco throttle">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(panel).png" height="300" alt="An interactive panel">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(iframes).png" height="300" alt=
+      "Multi throttle screens in iframes">
+      </p>
+
+      <p>The following setup will be handled in two sections, one for local internal streaming and control, and the next for streaming to remote engineers on the internet. Local streaming is good for testing or if you end up with too many engineers for a room, but not for the layout itself. Internal setup is useful to do before, as much of the setup applies to remote engineers on the internet.</p>
+      <p>You must also have every engine you want to run with the CabCam added to JMRI's roster. You can get as fancy or as spartan as you want with this, you can just add <strong>NMRA standard CV definitions</strong> decoders to your roster, the important bit is the DCC address is in the roster so it shows up in WebThrottle.</p>
+      <p><strong>Internal Setup</strong>First you are going to need to get the streaming URL for your CabCam which can be obtained from the <a href="https://siliconvalleylines.com/2022/08/11/cabcam-goes-uv4l/" target="_blank" rel="noreferrer noopener">CabCam setup instructions.</a> Setting the Memory Variables. Open Tools&gt;Tables&gt;Memory Variables.</p>
+      
+      <p><img src="images/JMRI_webThrottle-(tools).png" alt="JMRI Tools Menu" /></p>
+
+      <p>When this opens click <strong>Add... </strong>and a window will pop up. In <strong>System Name</strong> you want to enter I<strong>MWTVIDEOURL:XXXX </strong>where XXXX is the address of the engine you are wanting to run with the camera. For instance if I had UP 6936 in my roster I would add <strong>IMWTVIDEOURL:6936</strong>, and if I also wanted to run BNSF 326 I would add IMWTVIDEOURL:326 to the System Name. Don't enter anything for User Name, hit <strong>Create </strong>when done. If you have multiple entries to create you can just keep adding DCC addresses without closing that window by just changing the number at the end. </p>
+      </p>
```

Comment:
This is an extra close tag.

Reply:
Thanks, I am going to leave it in there for a bit, see if anyone else catches more I have missed so I can change it all at once. I am going to go through it more in depth again later tonight.